ray-operator/controllers/ray/utils/dashboard_httpclient.go

```diff
@@ -457,34 +407,3 @@ func (r *RayDashboardClient) DeleteJob(ctx context.Context, jobName string) erro
 
 	return nil
 }
-
-func ConvertRayJobToReq(rayJob *rayv1.RayJob) (*RayJobRequest, error) {
```

Comment:
any reason to move this function to `util.go`? It seems only to be used by `dashboard_httpclient.go`. We can also make it be a private function instead.